manifest.json

```diff
@@ -1,12 +1,12 @@
 {
 "name": "DuckDuckGo - Speed Dial",
 "developer": {"name" : "DuckDuckGo", "url" : "http://duckduckgo.com/?t=oen"},
-"description" : "Shows random Zero-click Info from DuckDuckGo in a Speed Dial window.",
+"description" : "Shows DuckDuckGo in a Speed Dial window.",
 "manifest_version" : 2,
 "version" : "1.3.1",
 "background" : {"page" : "index.html"},
 "icons" : {"128": "images/icon_64.png"},
-"web_accessible_resources" : [".gitignore","images/icon_64.png","images/logo.svg","readme.md","js/jquery.min.js","js/background.js","style.css"],
+"web_accessible_resources" : [".gitignore","images/icon_64.png","images/logo.svg","images/small_background.png","images/large_background.png","readme.md","style.css"],
```

Comment:
@mintsoft I am not sure why `.gitignore` has to be here, but I guess it does not hurt...